### PR TITLE
fix tests after lavaan update

### PR DESCRIPTION
**Pull request by Kucharssim**
see https://github.com/jasp-stats/jaspFactor/pull/149

### CODE REVIEW

File: R/sem.R [hunk 1]
```diff
@@ -509,6 +509,8 @@ checkLavaanModel <- function(model, availableVars) {
 
   # add test statistic correction footnote
   test <- lavaan::lavInspect(semResults[[1]], "options")[["test"]]
+  if(length(test) > 1)
+    test <- test[[2]]
```
Comment (juliuspf) [inline R/sem.R:513]: was there a bug?

**Review by juliuspf — CHANGES_REQUESTED** [review]
Well, it looks good, but ubuntu tests fail...

**Comment by juliuspf** [comment]
lets see if re-running helps

**Comment by shun2wang** [comment]
Don't worry, it's a matter of upstream action with a mirror server network error, we just have to wait for it.

File: R/sem.R (hunk 1 above)
Comment (Kucharssim) [reply R/sem.R:513]: seems like just an update, the `[["options"]][["test"]]` from `lavInspect` now always contains `standard` as the first entry regardless of the settings.

Comment (juliuspf) [reply R/sem.R:513]: that is not what I see when I run this
```r
library(lavaan)
packageVersion("lavaan") # 0.6.13

HS.model <- ' visual  =~ x1 + x2 + x3 
              textual =~ x4 + x5 + x6
              speed   =~ x7 + x8 + x9 '

fit <- cfa(HS.model, data = HolzingerSwineford1939, estimator = "MLMVS")
lavInspect(fit, "options")[["test"]]
```

Comment (Kucharssim) [reply R/sem.R:513]: True, but in that case `length(test)` is still 1, so the code does not do anything different to how it was before the change.

The double entries in the list is when you use the `test` argument:

```r
library(lavaan)
packageVersion("lavaan") # 0.6.13

HS.model <- ' visual  =~ x1 + x2 + x3 
              textual =~ x4 + x5 + x6
              speed   =~ x7 + x8 + x9 '

fit <- cfa(HS.model, data = HolzingerSwineford1939, test = "scaled.shifted")
lavInspect(fit, "options")[["test"]]
```

Comment (juliuspf) [reply R/sem.R:513]: you are absolutely right :)

**Review by juliuspf — APPROVED** [review]

**Pull request merged**